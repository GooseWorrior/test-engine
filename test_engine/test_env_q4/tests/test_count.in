newDefault 1
count 1 0
[]= 1 0 0
count 1 0 
count 1 1
[]= 1 1 1
count 1 0
count 1 1 
count 1 -1
erase 1 1
count 1 1
count 1 0
erase 1 0
count 1 0
[]= 1 5 5 
[]= 1 3 3 
[]= 1 4 4 
[]= 1 2 2
[]= 1 1 1
[]= 1 7 7
[]= 1 6 6
[]= 1 8 8
[]= 1 9 9
count 1 0
count 1 1
count 1 2
count 1 3
count 1 4 
count 1 5 
count 1 6 
count 1 7
count 1 8
count 1 9
count 1 10
newInit 2
count 2 1
count 2 0
count 2 7
count 2 -1
