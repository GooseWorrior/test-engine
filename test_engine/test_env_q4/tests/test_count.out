1:
0
1:
1: (0,0)
1
1: (0,0)
0
1: (0,0)
1: (0,0) (1,1)
1
1: (0,0) (1,1)
1
1: (0,0) (1,1)
0
1: (0,0) (1,1)
1: (0,0)
0
1: (0,0)
1
1: (0,0)
1:
0
1:
1: (5,5)
1: (3,3) (5,5)
1: (3,3) (4,4) (5,5)
1: (2,2) (3,3) (4,4) (5,5)
1: (1,1) (2,2) (3,3) (4,4) (5,5)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (7,7)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
0
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
0
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
2: (1,2) (3,4) (5,6) (7,8)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
2: (1,2) (3,4) (5,6) (7,8)
0
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
2: (1,2) (3,4) (5,6) (7,8)
1
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
2: (1,2) (3,4) (5,6) (7,8)
0
1: (1,1) (2,2) (3,3) (4,4) (5,5) (6,6) (7,7) (8,8) (9,9)
2: (1,2) (3,4) (5,6) (7,8)
